doc/nrf/ug_matter_creating_accessory.rst

```diff
@@ -7,24 +7,52 @@ Creating Matter accessory device
    :local:
    :depth: 2
 
+As part of this guide, you will modify the :ref:`Matter template <matter_template_sample>` sample to create a Matter accessory device that measures temperature and can be turned on and off.
```

Comment:
I think in the glossary we already use the term "Matter device" instead of "Matter accessory device" since Matter community avoids using the "accessory" term. Maybe we can start using "Matter device" here, too?

Reply:
Edited